opensearch/services/datasets/datasets.php

```diff
@@ -162,6 +178,19 @@ public function getSearchResultOutput(array $result) {
 			$html .= "</span>";
 		}
 
+		//data types
```

Comment:
This sounds like an expensive process performance wise. We need to take a look at how we might be able to cache this information or do something where we don't have to run a query per dataset in the result set.